src/main/kotlin/org/elm/ide/intentions/AddImportIntention.kt

```diff
@@ -81,16 +81,16 @@ class AddImportIntention : ElmAtCaretIntentionActionBase<AddImportIntention.Cont
                 // but if it's an alias-based import, we should show the user some UI so that
                 // they know what they're getting into. See https://github.com/klazuka/intellij-elm/issues/309
                 when {
-                    candidate.moduleAlias != null -> promptToSelectCandidate(context, file)
+                    candidate.moduleAlias != null -> promptToSelectCandidate(project, context, file)
                     else -> addImport(candidate, file, context.isQualified)
                 }
             }
-            else -> promptToSelectCandidate(context, file)
+            else -> promptToSelectCandidate(project, context, file)
         }
     }
 
 
-    private fun promptToSelectCandidate(context: Context, file: ElmFile) {
+    private fun promptToSelectCandidate(project: Project, context: Context, file: ElmFile) {
```

Comment:
Why add `project` parameter? It can be derived from the `file` parameter (or alternatively put it in the `Context`)

Reply:
Just because we already have it, so there's no need to derive. It's less code overall. I can revert this if you want.

Reply:
I guess my preference is always less-code-to-read vs less-code-to-execute. But I don't care either way here. I was just curious.